frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -538,6 +543,7 @@ const PairElement_ = DragSource(
                     <Button
                       type="button"
                       data-test-id="pairs-list__delete-btn"
```

Comment:
Same comment here. We should avoid having multiple test IDs.

Reply:
removed old test id